diktat-rules/src/test/resources/test/paragraph3/preview_annotation/PreviewAnnotationPrivateModifierExpected.kt

```diff
@@ -0,0 +1,20 @@
+package test.paragraph3.preview_annotation
+
+@Preview
+@Composable
+private fun BannerPreview1() {}
+
+@Preview
+private fun BannerPreview2() {}
+
+@Preview
+private fun BannerPreview3() {}
+
+@Preview
+private fun BannerPreview4() {}
+
+@Preview
+open fun BannerPreview5() {}
```

Comment:
why do we not change open?

Reply:
Probably you are right, will change this too